dist/milo_ui.bundle.js

```diff
@@ -2322,7 +2322,8 @@ function MLTime_get() {
     var hours = match[1]
         , mins = match[2];
     if (hours > 23 || mins > 59) return;
-    var time = new Date(1970, 0, 1, hours, mins);
+
+    var time = this.utc ? new Date(Date.UTC(1970, 0, 1, hours, mins)) : new Date(1970, 0, 1, hours, mins);
```

Comment:
If `utc` is enabled, the date object are instantiated to the provided hours/mins in UTC time, rather than local